source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1013,6 +1023,15 @@ namespace
 
 } // end anonymous namespace
 
+RtReadOptions::RtReadOptions() :
+    skipConflictingElements(true),
+    readFilter(nullptr),
+    readLookInformation(false),
+    desiredMajorVersion(MATERIALX_MAJOR_VERSION),
+    desiredMinorVersion(MATERIALX_MINOR_VERSION + 1)
```

Comment:
Moved from header file to replace magic numbers with build values. The +1 is still magic though...

Reply:
I'm not sure if the +1 is safe once we bump our version to 38, but no worse than hard-coding 38.